### PR TITLE
E_CLASSROOM-204 [FE/BE] Make Category Quizzes page dynamic

### PR DESCRIPTION
**Pull request by CaneteJoash**
### Issue Link
https://framgiaph.backlog.com/view/E_CLASSROOM-204

### Definition of Done
- [x] User will be able to see attempts of quiz dynamically

### Commands to Run
N/A

#### Main note

I have frontend pr.
- link: https://github.com/framgia/sph-classroom-els-fe/pull/61
#### Pages Affected
- `http://localhost:82/api/v1/QuizeAttempts/1`

#### Scenarios/ Test Cases
- [] it should `expected behavior` when `condition` 

### Screenshots
image

### CODE REVIEW

**Comment by Johnpaullim** [comment]
### Working 

- [x] User able to see attempts of quiz dynamically
- [x]  Able to Improvement the category quizzes page dynamic

<details><summary>Screenshot</summary>

Screenshot 2021-12-14 182748

Screenshot 2021-12-14 184044

Screenshot 2021-12-14 184123

</details> 

- Total Test time: 0.5 hr

File: app/Http/Controllers/API/v1/User/UserScoresAndAttemptsController.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+class UserScoresAndAttemptsController extends Controller
+{
+    public function UserScoreAndAttempts(Request $request)
+    {
+        $id = Auth::user()->id;
+
+        $recent = Quiz::join('quizzes_taken', 'quizzes.id', '=',  'quizzes_taken.quiz_id',)
+            ->where('quizzes_taken.user_id', '=', $id)
+            ->where('quizzes_taken.quiz_id', '=', $request->quiz_id)
```
Comment (RunstyleX) [inline app/Http/Controllers/API/v1/User/UserScoresAndAttemptsController.php]: We dont need the `=` sign here. but its okay to have. However, please update to remove

File: routes/api.php [hunk 2]
```diff
@@ -58,5 +59,6 @@
         Route::get('/followedLog', [StudentActivitiesController::class, 'followedActivities']);
 
         Route::get('/friendscore/{quiz_id}', [UserfriendsScoreController::class, 'friendsScore']);
+        Route::get('/QuizeAttempts/{quiz_id}', [UserScoresAndAttemptsController::class, 'UserScoreAndAttempts']);
```
Comment (RunstyleX) [inline routes/api.php]: Please fix type QuizAttempts

File: routes/api.php [hunk 3]
```diff
@@ -58,5 +59,6 @@
         Route::get('/followedLog', [StudentActivitiesController::class, 'followedActivities']);
 
         Route::get('/friendscore/{quiz_id}', [UserfriendsScoreController::class, 'friendsScore']);
+        Route::get('/QuizAttempts/{quiz_id}', [UserScoresAndAttemptsController::class, 'UserScoreAndAttempts']);
```
Comment (RunstyleX) [inline routes/api.php]:
```suggestion
        Route::get('/quiz_attempts/{quiz_id}', [UserScoresAndAttemptsController::class, 'UserScoreAndAttempts']);
```

**Pull request merged**